### PR TITLE
4.5.2 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-4.5.2-release](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-4.5.2-release)

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "@skyux/config": "4.4.0",
     "@skyux/core": "4.5.0",
     "@skyux/docs-tools": "4.9.0",
-    "@skyux/forms": "4.17.0",
+    "@skyux/forms": "4.17.1",
```
Comment (blackbaud-johnly) [inline package.json:58]: Don't we need to update the `popovers` version in line 3 of this file?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.5.2 (2021-05-10)
+
+- Fixed the dropdown component to focus on the first menu item when clicking the dropdown button. [#148](https://github.com/blackbaud/skyux-popovers/pull/148)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the dropdown component to focus on the first menu item after users select the dropdown button. [#148](https://github.com/blackbaud/skyux-popovers/pull/148)
```

File: package.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: Oops, didn't commit all my files. Good catch.

**Pull request merged**